src/libraries/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -616,12 +623,15 @@ internal abstract partial class SafeDeleteContext : SafeHandle
                     outContext._EffectiveCredential?.DangerousRelease();
                     outContext._EffectiveCredential = inCredentials;
```

Comment:
What happens if inCredentials is actually invalid because inCredentials.DangerousAddRef(ref mustReleaseCredentials) threw an exception?

(Same question for the similar case below)

Reply:
Then I expect this branch to not be taken because `errorCode` was initialized to `InvalidHandle`

https://github.com/rzikm/dotnet-runtime/blob/9cbe4b5191ab1679b12557e3c6830477952dd47c/src/libraries/Common/src/Interop/Windows/SChannel/Interop.SECURITY_STATUS.cs#L19